boards/arm64/intel_socfpga_agilex_socdk/intel_socfpga_agilex_socdk.dts

```diff
@@ -27,3 +27,7 @@
 	status = "okay";
 	current-speed = <115200>;
 };
+
+&qspi {
+	status = "okay";
```

Comment:
What about the flash device that is connected on the QSPI?  That should be described here.

Reply:
Here's an example:

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/arm64/boot/dts/intel/socfpga_n5x_socdk.dts#n83

Reply:
Hi Galak, 

This driver was ported from arm-trusted-firmware.
https://github.com/ARM-software/arm-trusted-firmware/blob/master/plat/intel/soc/common/drivers/qspi/cadence_qspi.c

https://github.com/ARM-software/arm-trusted-firmware/blob/master/plat/intel/soc/common/drivers/qspi/cadence_qspi.h

And there is no support for the dts to read from the connected device yet, while this driver generally allowed the raw access to read write at the cadence qspi flash and fully tested at our side, which is enough to support our another FPGA feature that we that will be up streamed soon.

Furthermore, due to the license issue, I believe we are not allowed to copy the driver code from the GPL in this case (Linux). Can we just let this changes go in first at least, and we will enhance it later?

Reply:
I think i got what you mean, i refer to https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nios2/altera_max10/altera_max10.dts

Reply:
@ngboonkhai - yeah, I assume you have a specific QSPI Flash module (Micron N25Q).   So I'm mostly asking to add that proper flash module in the devicetree dts for the board.  I'm ok with the fact that the driver only supports talking to that specific flash module.  (Some of the Kconfig params might be able to come from devicetree instead).

See what I did here:

commit 8d2cbc639c96862d44626f0cecd3d1f602b1edaf
and 
commit 78745b2de9e01328ae44e3a2c2e853dc1b25ddcc

Reply:
Hrmm, I actually not quite sure for the Kconfig option which are specific to the current flash, I currently don't have other flash with me to test with.  do you have any suggestion for that?

Reply:
It was a thought, if its not feasible right now that is fine.

Reply:
Yup, then i will close this as resolved. Thanks